common-lib/src/cmr/common/log.clj

```diff
@@ -32,6 +34,30 @@
             (str "\n" (t/stacktrace err {:stacktrace-fonts {}}))
             "")))
 
+;; The next two functions are needed to allow run time logging. The base
+;; knowledge for these functions came from here:
+;; https://github.com/ptaoussanis/timbre/issues/208
+;; https://github.com/yonatane/timbre-ns-pattern-level
+;; I modified the above solutions to be able to set a log level by namespace
+;; during run time.
+
+(defn ns-filter [fltr] (-> fltr enc/compile-ns-filter enc/memoize_))
+
+(defn log-by-ns-pattern
+  [& [{:keys [?ns-str config level] :as opts}]]
+  (let [ns-pattern-map (get config :ns-pattern-map)
+        namesp   (or (some->> ns-pattern-map
```

Comment:
That's a bit much, logic-wise, for a `let` block -- can you split this out into a private function (with docstring 😉)

Reply:
done.